x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -100,29 +100,39 @@ const defaultPostPushToService = {
   pushCaseToExternalService,
 };
 
-const fillFormReactTestingLib = async (renderResult: RenderResult) => {
+const sampleDataWithoutTags = {
+  ...sampleData,
+  tags: [],
+};
+
+const fillFormReactTestingLib = async (renderResult: RenderResult, withTags = false) => {
   const titleInput = within(renderResult.getByTestId('caseTitle')).getByTestId('input');
 
-  userEvent.type(titleInput, sampleData.title);
+  userEvent.paste(titleInput, sampleDataWithoutTags.title);
 
-  const descriptionInput = renderResult.container.querySelector(
-    `[data-test-subj="caseDescription"] textarea`
+  const descriptionInput = within(renderResult.getByTestId('caseDescription')).getByTestId(
+    'euiMarkdownEditorTextArea'
   );
 
-  if (descriptionInput) {
-    userEvent.type(descriptionInput, sampleData.description);
-  }
+  userEvent.paste(descriptionInput, sampleDataWithoutTags.description);
 
-  const caseTags = renderResult.getByTestId('caseTags');
+  if (withTags) {
+    const caseTags = renderResult.getByTestId('caseTags');
 
-  for (let i = 0; i < sampleTags.length; i++) {
-    const tagsInput = await within(caseTags).findByTestId('comboBoxInput');
-    userEvent.type(tagsInput, `${sampleTags[i]}{enter}`);
+    for (let i = 0; i < sampleTags.length; i++) {
+      const tagsInput = await within(caseTags).findByTestId('comboBoxInput');
+      userEvent.type(tagsInput, `${sampleTags[i]}{enter}`);
+    }
   }
 };
 
-// FLAKY: https://github.com/elastic/kibana/issues/142283
-describe.skip('Create case', () => {
+const waitForFormToRender = async (renderResult: RenderResult) => {
+  await waitFor(() => {
+    expect(renderResult.getByTestId('caseTitle')).toBeTruthy();
```

Comment:
nit: It sounds like the general advice is to start using `screen`. Using screen would also avoid having to pass in the `renderResult` parameter too

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
You are right!